theano/compile/tests/test_debugprint.py

```diff
@@ -0,0 +1,32 @@
+import theano
```

Comment:
Why did you add this file and this test? This is normally tested in theano/tests/test_printing.py